### PR TITLE
Added 2 READMEs to gh-pages branch

### PR DESCRIPTION
**Pull request by smileytechguy**
Created a README as well as some basic documentation on how to create a custom GUI

### CODE REVIEW

**Comment by smileytechguy** [comment]
Will reference in the develop branch README soon

**Comment by smileytechguy** [comment]
Added the 0.5.0-alpha4 theme to list

**Comment by smileytechguy** [comment]
Added my GUI with a better map bar to list

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+As a lot of people did not like the current web GUI, it is now de-centralized meaning that you can use anyone's GUI from the web!
+
+## Creation
+
+See DOCS.md for information on how to create a web-based GUI to share with others
+
+## User-created GUIs
+
+### [Theme from 0.5.0-alpha4 by @jabbink](http://pogo.abb.ink/0.5.0-alpha4/map.html)
```
Comment (jabbink) [inline README.md]: made by @shilch

**Comment by jabbink** [comment]
Remove the `.DS_Store` please. Also when you rename that screenshot of the current UI, it has to be edited in the `develop` branch as well.

**Comment by jabbink** [comment]
Also the other UI references to some version with a Pokevision thing.

1. Pokevision is offline
2. That service is entirely useless as this bot also has information of surrounding Pokemon. Just make the socket server the info to the UI and render it in a way so that the user can see which are caught and which aren't.

**Comment by smileytechguy** [comment]
For my theme i messed up the commit history, that button is disabled.  That theme was there due to map links and easy coordinate access. I copied the 0.5.4 screenshot into the gui-screenshot as well.

**Pull request merged**